src/Themes/NexusMods.Themes.NexusFluentDark/Styles/Controls/Button/StandardButtonStyles.axaml

```diff
@@ -205,21 +211,17 @@
             <!-- NOTE: Spacing is handled through margin of icon and images as icons seem to behave strangely -->
         </Style>
 
-        <Style Selector="^ > StackPanel > avalonia|Icon">
-            <!-- Icons size seem determined by font size, width and height are just there to limit the bounds -->
-            <Setter Property="FontSize" Value="24" />
-            <Setter Property="Width" Value="24" />
-            <Setter Property="Height" Value="24" />
+        <Style Selector="^ > StackPanel > unifiedIcon|UnifiedIcon">
+            <Setter Property="Size" Value="24" />
 
             <!-- NOTE: For some reason setting this to 8 causes a big distance for icons only, this is a workaround -->
             <!-- If the distance from the text is too low, try setting this to 8 as design -->
             <Setter Property="Margin" Value="0,0,3,0" />
         </Style>
 
-        <Style Selector="^ > StackPanel > Image">
+        <Style Selector="^ > StackPanel > unifiedIcon|UnifiedIcon">
             <Setter Property="Margin" Value="0,0,8,0" />
-            <Setter Property="Width" Value="24" />
-            <Setter Property="Height" Value="24" />
+            <Setter Property="Size" Value="24" />
```

Comment:
We have two styles here for UnifiedIcon.
Basically I noticed that there was a difference in the margin when an image was used rather than an icon and tried to account for that. 

The supposed margin should have been 8 (should actually have been `StackPanel.Spacing`) but the final result was a bigger gap than what was on figma.

For now set the spacing to 8 and remove the double style